src/SFA.DAS.AssessorService.ExternalApiDataSync/ExternalApiDataSyncCommand.cs

```diff
@@ -17,13 +24,26 @@ public class ExternalApiDataSyncCommand : ICommand
         private readonly string _sourceConnectionString;
         private readonly string _destinationConnectionString;
 
+        private readonly BulkCopySettings _bulkCopySettings;
+
         public ExternalApiDataSyncCommand(IWebConfiguration config, IAggregateLogger aggregateLogger)
         {
+            // TODO: When we're allowed to do changes that affect Staff UI...
```

Comment:
This should be a ticket with the number referenced here

Reply:
Added ticket numbers